components/notebook-controller/config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: IMAGE_URL
+      - image: gcr.io/kubeflow-images-public/notebook-controller:v20201122-v1.2-rc.0-5-g87c6d9c0-dirty-1a1de6
```

Comment:
It's not necessary to commit this change. Unfortunately, making the branch dirty after a test is a side-effect of kustomize not allowing configuration with envvars, because everything should be recorded in git. So you can leave this one, maybe we can make this experience better in the future 🤔